api/graphql/reservation_units/reservation_unit_filtersets.py

```diff
@@ -62,16 +62,36 @@ def get_text_search(self, qs, property, value: str):
         queries = []
         for word in words:
             queries.append(
-                Q(name__icontains=word)
-                | Q(description__icontains=word)
-                | Q(spaces__name__icontains=word)
-                | Q(keyword_groups__name__icontains=word)
-                | Q(resources__name__icontains=word)
-                | Q(services__name__icontains=word)
-                | Q(purposes__name__icontains=word)
-                | Q(reservation_unit_type__name__icontains=word)
-                | Q(equipments__name__icontains=word)
-                | Q(unit__name__icontains=word)
+                Q(name_fi__icontains=word)
+                | Q(name_en__icontains=word)
+                | Q(name_sv__icontains=word)
+                | Q(description_fi__icontains=word)
+                | Q(description_en__icontains=word)
+                | Q(description_sv__icontains=word)
+                | Q(spaces__name_fi__icontains=word)
+                | Q(spaces__name_en__icontains=word)
+                | Q(spaces__name_sv__icontains=word)
+                | Q(keyword_groups__name_fi__icontains=word)
+                | Q(keyword_groups__name_en__icontains=word)
+                | Q(keyword_groups__name_sv__icontains=word)
+                | Q(resources__name_fi__icontains=word)
+                | Q(resources__name_en__icontains=word)
+                | Q(resources__name_sv__icontains=word)
+                | Q(services__name_fi__icontains=word)
+                | Q(services__name_en__icontains=word)
+                | Q(services__name_sv__icontains=word)
+                | Q(purposes__name_fi__icontains=word)
+                | Q(purposes__name_en__icontains=word)
+                | Q(purposes__name_sv__icontains=word)
+                | Q(reservation_unit_type__name_fi__icontains=word)
+                | Q(reservation_unit_type__name_en__icontains=word)
+                | Q(reservation_unit_type__name_sv__icontains=word)
+                | Q(equipments__name_fi__icontains=word)
+                | Q(equipments__name_en__icontains=word)
+                | Q(equipments__name_sv__icontains=word)
+                | Q(unit__name_fi__icontains=word)
+                | Q(unit__name_en__icontains=word)
+                | Q(unit__name_sv__icontains=word)
```

Comment:
I wonder if it would be possible to fetch all the translated fields dynamically and then construct the filter in e.g. a `for` loop. 🤔 But the current solution is okay too. 👍

Reply:
yes it would, using `get_translatable_fields` perhaps. I'll check :+1:

Reply:
well, we could do something dynamic but that would perhaps make nonsense since most of the `Q`'s are focused on fk/m2m relations it would be too complex (imo) to proceed with that. What we currently have is that we can find out the model's translated fields and translated field of given model's specific field. But using those we need to know the models so we would need to specify nine (9) model relations and make'em loop etc... Perhaps you had something else in mind but at least this was my conclusion of this :upside_down_face: